### PR TITLE
Update expectations for other.test_metadce_minimal_pthreads after #15604

### PR DESCRIPTION
**Pull request by sbc100**
This broke the emscripten-releases roller as well as recent test
runs on emscripten CI.

I have no idea how it actually passed in #15604.  That is rather
worrying.

### CODE REVIEW

**Review by dschuff — APPROVED** [review]
LGTM, assuming this test change reflects the desired behavior of course :)

**Review by kripken — APPROVED** [review]

**Comment by kripken** [comment]
Perhaps it passed before due to a race in landing, with maybe #16129 ? If so, I'd expect it to fail not just on the roller but also on main here, only after landing, which as you said is indeed the case.

**Comment by sbc100** [comment]
Ah yes, that makes sense

**Comment by sbc100** [comment]
I'm going to land this TBR, once (maybe once the mac build confirms that the other suite is all good).. to get things green again.

**Comment by sbc100** [comment]
> I'm going to land this TBR, once (maybe once the mac build confirms that the other suite is all good).. to get things green again.

Sorry, not TBR, just land without waiting for all results.

**Pull request merged**
